apps/web-app/pages/join/profile.vue

```diff
@@ -229,11 +229,13 @@
             v-model:value="fNameValue"
             v-model:valid="fNameValid"
             label="First Name"
+            :schema="z.string().min(1).max(32).trim()"
```

Comment:
schema validation didnt exist here before
maybe we can just set it to z.string().trim() without a min/max